frontend/src/features/documents/list/DocumentListView.tsx

```diff
@@ -17,6 +18,7 @@ export interface IDocumentListViewProps {
   documentResults: Api_DocumentRelationship[];
   hideFilters?: boolean;
   defaultFilters?: IDocumentFilter;
+  deleteHandle: (relationship: Api_DocumentRelationship) => Promise<boolean>;
```

Comment:
nit: perhaps could name this "onDelete" to align with FE naming conventions? not a big deal